addons/account/views/account_report.xml

```diff
@@ -14,6 +14,18 @@
             groups="account.group_account_invoice"
         />
 
+        <report
+            id="action_account_original_vendor_bill"
+            model="account.invoice"
+            string="Original vendor bills"
+            report_type="qweb-pdf"
+            name="account.report_original_vendor_bill"
+            file="account.report_original_vendor_bill"
+            attachment="(object.state in ('open','in_payment','paid')) and ('INV'+(object.number or '').replace('/','')+'.pdf')"
```

Comment:
might want to change the name of this attachment (it does not really matter as it won't be saved but for comprehension).
Also you might want to change the print_report_name as this will be called if there is only one invoice selected and printed
I suggest:
```
attachment="original_vendor_bill.pdf"
print_report_name="(object._get_original_report_base_filename())" 
```
And create a new method _get_original_report_base_filename that would return the name of the original file